simulations/getting-started.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Getting Started
+---
+
+import { Tabs, Tab } from 'nextra/components';
+
+This guide will walk you through the basic setup required to run your first simulation and see the results in LangWatch.
+
+For more in-depth information and advanced use cases, please refer to the official [`scenario` library documentation](https://github.com/langwatch/scenario).
+
+## 1. Installation
+
+First, you need to install the `scenario` library in your project. Choose your language below.
+
+<Tabs>
+  <Tab title="Python">
+    ```bash
+    pip install scenario-langwatch
+    ```
+  </Tab>
+  <Tab title="TypeScript">
+    ```bash
+    npm install @langwatch/scenario
+    ```
+  </Tab>
+</Tabs>
+
+
+## 2. Configure Environment Variables
+
+We recommend creating a `.env` file in the root of your project to manage your environment variables.
+
+```bash title=".env"
+LANGWATCH_API_KEY="your-api-key"
+LANGWATCH_ENDPOINT="https://app.langwatch.ai"
+```
+
+You can find your `LANGWATCH_API_KEY` in your LangWatch project settings.
+
+
+## 3. Create a Basic Scenario
+
+Here's how to create and run a simple scenario to test an agent.
+
+First, you need to wrap your agent in a simple function that takes a list of messages and returns a string response.
+
+While optional, we strongly recommend setting stable identifiers for your scenarios, sets, and batches for better organization and tracking in LangWatch.
+
+-   **`id`**: A unique and stable identifier for your scenario. If not provided, it's often generated from the `name`, which can be brittle if you rename the test.
+-   **`setId`**: Groups related scenarios into a test suite. This corresponds to the "Simulation Set" in the UI.
+-   **`batchId`**: Groups all scenarios that were run together in a single execution (e.g., a single CI job). You can use a CI environment variable like `process.env.GITHUB_RUN_ID` for this.
```

Comment:
I actually would avoid mentioning all those options, if we don't force them from library side, its weird we would right away on the docs when they are just getting started

I would have that on a separate section about "Grouping your sets and batches" or something like that